src/main/java/com/saiko/bidmarket/user/service/DefaultUserService.java

```diff
@@ -176,6 +177,22 @@ public List<UserHeartSelectResponse> findAllUserHearts(
                             .collect(Collectors.toList());
   }
 
+  @Override
+  public UserHeartCheckResponse isUserHearts(
+      long userId,
+      long productId
+  ) {
+
+    productRepository
+        .findById(productId)
+        .orElseThrow(() -> new NotFoundException("Product does not exist"));
+
+    return heartRepository.findByUserIdAndProductId(userId, productId)
+        .map(Heart::isActived)
```

Comment:
```suggestion
    return heartRepository
        .findByUserIdAndProductId(userId, productId)
        .map(Heart::isActived)
```
이거 포메팅하면 한줄 넘어가지 않나요?

Reply:
오호 적용하겠ㅅ브니다